api/scm.go

```diff
@@ -107,6 +107,11 @@ func SyncRepos(c *gin.Context) {
 
 				return
 			}
+			e = updateUserFavorites(c, repo)
+			if e != nil {
```

Comment:
Can you update `e` -> `err` here?

```suggestion
			err = updateUserFavorites(c, repo)
			if err != nil {
```

In Go, it is commonplace and idiomatic to use `err` as a variable name to represent an `error`.